lib/budo.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
There was a flow error for not specifying the signature of the function in this file, but since we don't transpile these files there was an error when actually running the file.